pkg/standalone.js

```diff
@@ -1,5 +1,25 @@
-import init from "./standalone/content_tag.js";
+import init from "./node/content_tag.js";
```

Comment:
we don't want to use the `node` version of this because it's not at all compatible with browsers.

Likewise (and most annoyingly!) the standalone version also isn't compatible with node (I have a small hunch though that the standalone version _will eventually_ be compatible with node... but we'll see)

Reply:
the way you can test this is by booting up Vite (`pnpm start`) and see if the module loads and parses, etc.

(This is probably also a good way to spot check sourcemaps)

Reply:
Ahh my mistake, not sure why I did that probably copy/paste mistake when I was duplicating files.
It should be fixed now @NullVoxPopuli

Reply:
thanks!!